toptal/picasso
# style(TextEditor): [FX-2297] Add styling to texteditor component

[FX-2297](https://toptal-core.atlassian.net/browse/FX-2297)

[Abstract](https://share.goabstract.com/2de8e3f4-5704-4139-8a68-9e9519ed09f8?mode=build&sha=4f1f6493dfac89015cc6c71ea348807e931fe3bc)

### Description

- TextEditor looks like in BASE design
- TextEditor disabled looks like in BASE
- Toolbar looks like in BASE design
    - style options according to design (**new PR**)
- TextEditor height is resizable by dragging (**[new PR](https://github.com/toptal/picasso/pull/2353)**)

> NOTE:
>
> We will create separate tasks for:
> - styling options
> - making TextEditor resizable
> - rewriting logic for toolbar actions

### How to test

- Visit [Picasso Text Editor component](https://picasso.toptal.net/fx-2297-add-styling-to-texteditor-component/?66354e84639b6ae7e16e17ba5da6bde10caf6c31), hover and click on each toolbar action buttons, they should look exactly like in the design.

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-2297]: https://toptal-core.atlassian.net/browse/FX-2297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
